communities/citytechsocspring/community-assessment.md

```diff
@@ -0,0 +1,112 @@
+
+Yes
+- Are they trying to ramp-up to get a job?
+
+Members on from the Computer Science course do sometimes use the community to learn more about roles in tech and ask questions from older members.
+- Are they passionate about developing, regardless of their major?
+
+The students that we see from non CS degrees tend to be more passionate about developing whereas some of our CS students fail to make the distinction between coursework and working on their own projects and do sometimes see developing as a chore.
+- What activities and goals can you set for your community based on their existing goals?
+
+We are looking into working on a project together as a community, whether it be a website or game. This will be inviting to members who want to develop more as well as CS students who want to work on their portfolio. 
+
+### Location
+- Where does your community currently gather?
+
+We usually gather in a room in the CS department. We are looking to get a permanent space for meeting in the future.
+- Is there public transportation available to and from meet-ups?
+
+Students are usually already on campus for gatherings but our campus is in Central London so it can be accessed by the tube or bus easily. 
+- How much does it cost to use the space?
+
+Spaces on campus are free to use.
+- How flexible is the space for different types of presentations and meetings?
+
+Depending on the type of event being held we book a different room. So for weekly meetups or workshops we book a classroom but if we need more capacity we can book a lecture theatre. 
+- How many people can it fit?
+
+Our normal rooms can fit around 30 people, lecture theatres can fit upwards of 150 people.
+- Is there comfortable seating for all shapes and sizes?
+
+We tend to use classrooms which have more comfortable seating and a better set up for our events compared to a lecture theatre.
+- Is there water or snacks available nearby?
+
+Water is available nearby on campus and there are shops very close to the university. We also bring a small number of snacks to the meetups when we can and cater for larger events such as hack days. 
+
+### Communication
+- How does your community get in touch with one another?
+
+Right now we use the mailing list and slack for communication. 
+- Is there an email group, a GitHub organization, a Twitter account, a Facebook group, a chat room or channel?
+
+We have a GitHub org for committee members to manage the website. We have a public Twitter and Facebook page for announcements to external people and a currently inactive Facebook group.
+- Could your organization benefit from one or more of those, or another way of communicating?
+
+We are in discussions of a platform to move onto from Slack to get more engagement from members. 
+
+### Culture
+- How welcoming is your community to new people and new ideas?
+
+We try to be very welcoming; we tend to keep a flat structure as far as members are concerned so nobody is worried about speaking their mind.
```

Comment:
We tend to ask for feedback from members at the beginning of weekly meetups. Along the lines of what sort of activities do you want to see, what could use improvement etc.